test/utils/session-helper.test.js

```diff
@@ -31,73 +22,3 @@ describe('createResourceTtlFile', () => {
     expect(Object.keys(result.predicates)).toHaveLength(7);
   });
 });
-
-describe('getContainerUrl', () => {
```

Comment:
Will soon replace functions that still uses usernames with podUrl. Phasing out unit test for getContainerUrl and getPodUrl. Will phase these functions completely in future PR when we replace them for set permission forms.